web/src/pages/subscription-plans/index.page.jsx

```diff
@@ -90,6 +91,17 @@ const SubscriptionPlans = () => {
         component="section"
         sx={{ maxWidth: '1280px', margin: '0 auto' }}
       >
+        {currentSubscription
+              && currentSubscription.cancelAtPeriodEnd
+              && (
+              <Text>
+                You current subscription ends
+                {' '}
+                {dayjs(new Date(currentSubscription.endDate * 1000)).format('MMM DD, YYYY')}
```

Comment:
Please create variable for end subscription date and use string interpolation here